translations/en.json

```diff
@@ -2150,5 +2151,42 @@
     "user-shared-a-community": "{{user}} shared a community",
     "you-deleted-a-message": "You deleted a message",
     "this-message-was-deleted": "This message was deleted",
-    "user-deleted-a-message": "{{user}} deleted a message"
+    "user-deleted-a-message": "{{user}} deleted a message",
+    "view-channel-members-and-details": "View channel members and details",
+    "unmute-channel": "Unmute channel",
+    "share-channel": "Share channel",
+    "all-messages": "All messages",
+    "muted-until": "Muted until",
+    "until-you-turn-it-back-on": "you turn it back on",
+    "mon": "Mon",
```

Comment:
I always thought month abbreviations were provided by Google's localization lib. Are you sure this is needed?

Reply:
We don't know what language the user is using, So I think we need to hardcode at least English.
I'll try to think of a way without this

Reply:
Pardon my ignorance, I just thought we were respecting the OS's language in the app, never hardcoding English in the i18n setup.

My experience with i18n libraries were always that month names were already translated in all supported languages, as well as units, and so on. But may it's different in mobile, and I don't really remember now.

Reply:
I meant hardcoding the i18n keys and using them to be referenced in other languages, For example we hardcoded the variable months inside `constants.cljs` so that we can have a reference to the month name that won't change in i18n and it's translated value would be the one changing
I can do it as integers instead, For example
```clj (def months {1 :month-1 2 :month-2 3 :month-3})``` instead of putting in the english translation, Their job is to only index the month value to be correctly translated

Reply:
Thanks for the explanation.

What I mean is that the code below should be unnecessary, because i18n gives us localized formatters for dates (which includes all the names of weeks and months, abbreviated or not).

```clojure
(i18n/label
 (keyword "t"
          (get constants/months
               (t/month parsed-time))))
```

If what I'm saying is true, then we shouldn't use `i18n/label` for dates, otherwise we need to start adding manual translations for things that are already provided by i18n. I think the correct way is to just get the parsed time and format it using the pattern for months. There's no need to get the month and then fetch a custom translation.

Here are all the patterns:

https://github.com/google/closure-library/blob/master/closure/goog/i18n/datetimeformat.js#L67

Reply:
Thank you so much for looking at this more closely, I will see how I can refactor this after I finish the current task I am on which won't take long.

Reply:
Thank you @ibrkhalil, and sorry for not seeing this problem earlier.